### PR TITLE
[CR] Jewelry and Accessories Additions

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
```SUMMARY: Content "Adds new jewelry and accessories"```

#### Purpose of change
Jewelry is a very diverse area. There are many kinds of jewelry, and many kinds of materials to make them with. Aside from generic jewelry like gold rings, there is also special jewelry such as engagement rings.

In the game, there is several instances of jewelry kinds that only exist as a certain material, like the locket (only one kind exists in-game, made from silver), but other kinds of lockets such as gold and copper lockets also exist.

There are more kinds of ties than just the skinny tie, such as the bow tie and clip-on tie, and naming the tie "skinny tie" implies that there are more kinds in game, when there wasn't.

I was inspired to do this because one of my goals in the game was to collect two of every kind of jewelry, and I was a bit disappointed when I ran out of jewelry to collect.

#### Describe the solution
Adds a large amount of new jewelry to the game. I focused on three of the most common jewelry materials - platinum, silver and gold. I also added copper variants of some jewelry kinds, because copper jewelry already exists in the game (it doesn't have FANCY though). There was no platinum jewelry in the game, so I added platinum variants of the existing jewelry.

The following items had their generic item IDs changed to more specific ones, and were added to migration.json to edit existing items:
"ring" -> "gold_ring"
"necklace" -> "silver_necklace"
"brooch" -> "jade_brooch"
"locket" -> "silver_locket"
"skinny_tie" -> "tie_skinny"

It's easier to explain with a list of the added jewelry. Of the new jewelry added, only the Platinum Watch, Gold Medal and Platinum Dental Grill received the SUPER_FANCY tag. Most of the non-copper additions received FANCY.

**Existing Jewelry (different material variants of existing jewelry):**
Silver Watch
Silver Ring 
Gold Necklace
Gold Locket
Copper Necklace
Copper Ring 
Copper Locket
Beaded Earring 

**Platinum Variants of Existing Jewelry:**
Platinum Ring 
Platinum Bracelet 
Platinum Watch
Platinum Necklace 
Platinum Earring 
Platinum Locket
Platinum Dental Grill

**New Jewelry Kinds:**
Gold, Silver and Bronze Medal 
Platinum, Gold, Silver and Copper Hairpin 
Cufflink and Intricate Cufflink 

**Special Jewelry:**
Chastity Ring 
Wedding Ring 
Engagement Ring 
Signet Ring 
Friendship Bracelet 

**Accessories:**
Bow Tie
Clip-on Tie
Tie

#### Additional context
[CR] Because I have no idea what to do with FANCY and Stylish. Adding all this new jewelry arguably unbalances Stylish, making it easy to get the +25 bonus without wearing actual clothes like the maid outfit.

I'm not very good at coding, so I wasn't able to fix the problem. My suggestion to fix the issue is adding a new flag, PARTLY_FANCY, that gives 1/5 of a morale point. Then SUPER_FANCY jewelry can be given the FANCY flag instead and FANCY jewelry gets the new PARTLY_FANCY flag.

### CODE REVIEW

**Comment by Mecares** [comment]
You might want to use looks_like for those new items.

**Comment by SourSlime** [comment]
Thanks for reminding me about that, but I'm not sure how useful looks_like will be. The main problem is that each item/accessory has a different material (gold necklace vs silver necklace), or colour (clip-on tie vs necktie). For tilesets which don't have active updaters, that would be useful, but for ones like Dead/LivePeople, the artists are going to want different tiles for each one.

**Comment by SunshineDistillery** [comment]
There's a simpler way of doing this.  

You could instead make a smaller list of generic jewelry types, like copper_jewelry, gold_ring, plastic_necklace, etc.  And then once you have a reasonable set, use snippets to give multiple  descriptions for each item.  Gold ring could be described variously as an engagement ring, wedding ring, etc.

That would limit the number of items a character could wear, 1 or 2 each, and would cut down on the total number of items and recipes.

**Comment by SourSlime** [comment]
I’m not sure how to do that, but I don’t think it’s a good idea. Most people can wear up to 18 small rings (3 on 6 fingers over two hands) without causing any inconvieniences, and the player should be anble to wear a wedding ring, engagement ring and another ring at the same time.

Each piece of jewelry also has a different weight, different price and different material (I’m not sure if this is compatible with what you’re suggesting).

The issue I’m having is balance with Stylish. Afaik, Stylish is meant to give a morale boost by trading warmth, protection, etc. for stylish clothes, and adding all this new jewelry only compounded the existing problem.

**Comment by Mecares** [comment]
Just for clarification looks_like does not prevent individual tiles for an item it just defaults to the specified if it has no own tile in the tileset.

**Comment by Rivet-the-Zombie** [comment]
Are any of these craftable? I want to give friendship bracelets to all of my NPC buddies.

**Comment by Fris0uman** [comment]
The max mood bonus for stylish is already capped at 15 or 20, and it's already easy to get there with the available jewlry in the game. So I don't think giving FANCY to all the jewlery will make it worse.
This is a problem with the stylish trait itself and not with your work on additional jewlery i think.

File: data/json/items/armor.json [hunk 1]
```diff
@@ -7405,6 +7187,24 @@
...
   },
+  {
+    "id": "platinum_watch",
+    "type": "ARMOR",
+    "name": "platinum watch",
+    "name_plural": "platinum watches",
+    "description": "Flex on those gold-watch wearers with the rare, expensive and superior platinum watch.  This one is made from solid platinum.",
+    "weight": 80,
+    "volume": 1,
+   //far more expensive than gold watches, around $100,000 for most platinum watches
```
Comment (ZhilkinSerg) [inline data/json/items/armor.json]:
```suggestion
   "//": "far more expensive than gold watches, around $100,000 for most platinum watches",
```

Comment (SourSlime) [reply data/json/items/armor.json]: Thanks

**Comment by SourSlime** [comment]
@curstwist has resolved the merge conflicts in #32042, so I'm closing this.

**Pull request closed without merging**